src/applications/vaos/api/index.js

```diff
@@ -478,12 +478,28 @@ export function updateRequest(req) {
 export function submitRequest(type, request) {
   let promise;
   if (USE_MOCK_DATA) {
-    promise = Promise.resolve({
-      data: {
-        id: 'testing',
-        attributes: {},
-      },
-    });
+    if (request.typeOfCareId === EXPRESS_CARE) {
+      promise = Promise.resolve({
+        data: {
+          id: 'testing',
+          attributes: {
+            email: request.email,
```

Comment:
I'm assuming these field names based on the backend field names. We can tweak them as necessary when the form page is built.